### PR TITLE
Add app descriptions to root README

### PR DESCRIPTION
**Pull request by rcebulko**
See [the branch README](https://github.com/rcebulko/amp-github-apps/tree/docs) for the Markdown-rendered view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,50 @@
...
+
+The app runs on an instance of Google AppEngine and is installed exclusively on the `ampproject/amphtml` repository.
+
+### [Owners Bot](owners/README.md)
+
+Contact: [@rcebulko](https://github.com/rcebulko)
+
+A web service that suggests approvers for a GitHub pull request based on OWNERS files and enforces reviews by OWNERS as a GitHub status check.
+
+The app runs on an instance of [Google AppEngine](https://ampproject-owners-bot.appspot.com) and is installed exclusively on the `[ampproject/amphtml](https://github.com/ampproject/amphtml)` repository. It can be trivially deployed to other organizations and repositories.
```
Comment (rsimha) [inline README.md]: In order to create a link that uses the code font, you'll need to move the backticks to inside the square brackets here and everywhere else.

**Wrong:**
```
`[ampproject/amphtml](https://github.com/ampproject/amphtml)`
```
will generate `[ampproject/amphtml](https://github.com/ampproject/amphtml)`


**Correct:**
```
[`ampproject/amphtml`](https://github.com/ampproject/amphtml)`
```
will generate [`ampproject/amphtml`](https://github.com/ampproject/amphtml)

**Comment by estherkim** [comment]
PR Deploy section looks good.

File: README.md [hunk 2]
```diff
@@ -2,5 +2,50 @@
...
+
+### [PR Deploy Bot](pr-deploy/README.md)
+
+Contact: [@estherkim](https://github.com/estherkim)
+
+A GitHub App which deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1).
```
Comment (danielrozenberg) [inline README.md]: This URL is not publicly accessible

File: README.md [hunk 3]
```diff
@@ -2,5 +2,50 @@
 
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-This repository will contain the source code of GitHub Apps designed
+This repository contains the source code of GitHub Apps designed
```
[the remark this answers is not in the document]
Comment (rcebulko) [reply README.md]: Lol tells us how outdated this file was

File: README.md (hunk 2 above)
Comment (rcebulko) [reply README.md]: I pulled this directly from the PR Deploy Bot README, so I guess that should be updated too. I'm going to remove both hyperlinks in this PR unless @estherkim has a suggestion for something to replace it with.

File: README.md (hunk 1 above)
Comment (rcebulko) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -2,5 +2,50 @@
...
 specifically for the AMP Project.
+
+## Apps
+
+### [Bundle-size Bot](bundle-size/README.md)
+
+Contact: [@danielrozenberg](https://github.com/danielrozenberg)
```
[the remark this answers is not in the document]
Comment (rcebulko) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (estherkim) [reply README.md]: Removing it is fine with me. Thanks.

**Pull request merged**